src/lib/booleans.js

```diff
@@ -0,0 +1,116 @@
+function negate(a) {
+  return !a;
+}
+
+function both(a, b) {
+  return a && b;
+}
+
+function either(a, b) {
+  return a || b;
+}
+
+function none(a, b) {
+  if (a === false && b === false) {
+    return true;
+  }
+
+  return false;
+}
+
+function one(a, b) {
+  return Boolean(a ^ b);
+  // NEED TO REVIEW THE ABOVE*//
+}
+
+function truthiness(a) {
+  return Boolean(a);
+}
+
+function isEqual(a, b) {
+  if (a === b) {
+    return true;
+  }
+
+  return false;
```

Comment:
You can update a lot of these functions to be more concise by simply returning the evaluation of the condition, for example here:
```
return (a === b)
```
If it evaluates to `true`, you get `true` returned and vice versa